added content to 'become an open science champion' chapter

ASarafoglou opened the pull request:
The chapter is not completed yet, but I guess it is a good start. Let me know what you think.

mgalland committed:
make intro message for alex

Commit:
include text for 'becoming an open science champion'

I added the following paragraphs:

- the crisis of confidence (what it is and what the cause is)
- open research pracitces (i.e., data sharing, preregistration, reproducible analyses)
- some ideas for in-class assignments

mgalland commented:
Hello Alexandra, 
Thanks for your great input. It already looks pretty good in terms of content and length!

## Additional info
- Attendees will be exclusively Life Scientists (from SILS and IBED from the Faculty of Science).
- We expect ~20 participants, not too small not too big. 
- They range from Master students to Principal Investigators (no professors yet!) 

## Comments on the "Crisis of Confidence" section
This paragraph is perfectly in the scope of that workshop section. 

## Comments on "Open and Reproducible Research Practices" section
- On data sharing, this is essential. Although I do it mostly on Github for small size datasets, Zenodo (CERN archive) is also a great resource. The UvA has also a Figshare partnership. Maybe we could give the audience some more practical tips there.
- On the reproducibility of statistical results, perhaps the term I would use is "Reproducible data workflow" which would actually make a point on why to have RMarkdown documents for instance. The p-value issue is also interesting (misuse of it).
- I think Preregistration and Registered Reports are specific to your field (psychology right?). I don't see any example for Biology but maybe you can prove me wrong?

That's all for now from me. I'll have to take a second look at it next week hopefully. Let me know what you think! 

Some suggestions:
1. Could you perhaps rephrase "The Crisis of Confidence" into "The Reproducibility Crisis"? And write accordingly? The main message would be to show the importance and improvements associated with opening data and code.

mgalland committed:
make small change to main text, add links

Commit:
Incorporated Marcs comments

- provided example for registered reports in the field of cancer biology
- rephrased crisis of confidence to reproducibilty crisis
- added OSCA as idea for community
- added public data sharing repositories and the RDM support of the UvA

ASarafoglou committed:
Merge branch 'gh-pages' into alex

ASarafoglou commented:
Hi Marc,

I incorporated your changes now in our document. I am not a data sharing expert (especially not in disciplines other than psychology), so I just added some UvA resources on the topic, that is, I linked to the research data management support team. Do you think an exercise related to data sharing would be helpful? For instance, a tutorial on how to upload and share data on Zenodo?

mgalland committed:
Fix typos and reshuffled the Open Science section

Placed the Open Science citation at the top. 
Fixed typos in the text.

ASarafoglou committed:
Power Point Presentation for OSCA Workshop

ASarafoglou committed:
added analysis blinding and sensitivity analyses to document

ASarafoglou committed:
added link to workshop slides

mgalland commented:
Merged locally into the gh-pages branch.
